src/test/kotlin/no/nav/syfo/personstatus/api/v2/DialogmotekandidatPersonoversiktStatusApiV2Spek.kt

```diff
@@ -124,6 +143,20 @@ object DialogmotekandidatPersonoversiktStatusApiV2Spek : Spek({
     }
 })
 
+fun createPersonoversiktStatusWithTilfelle(database: TestDatabase) {
+    val kafkaOppfolgingstilfellePerson = generateKafkaOppfolgingstilfellePerson(
+        personIdent = PersonIdent(ARBEIDSTAKER_FNR),
+        virksomhetsnummerList = listOf(
+            UserConstants.VIRKSOMHETSNUMMER_DEFAULT,
+            Virksomhetsnummer(UserConstants.VIRKSOMHETSNUMMER_2),
```

Comment:
Ikke knyttet til denne PRen, men burde vi være konsekvente med om testdataene våre skal være String eller objekter? 🤔

Reply:
Gjerne for min del. Men hvilken skal vi velge?

Reply:
Er det best med objekt? Det gir jo mer typesikkerhet 🤔

Reply:
Tror det